platform/jewel/markdown/core/src/test/kotlin/org/jetbrains/jewel/markdown/processing/MarkdownProcessorOptimizeEditsTest.kt

```diff
@@ -1023,6 +1025,425 @@ public class MarkdownProcessorOptimizeEditsTest {
         )
     }
 
+    @Test
+    public fun `regression JEWEL-819 put the a row in before empty lines`() {
```

Comment:
Not sure what "the a row" means? Should it be "a new row"?

Reply:
Also, is this different from the last new test? Looks identical at first glance

Reply:
No, I think I moved it to the first commit but somehow it was duplicated.